Lib/test/test_asyncio/test_tasks.py

```diff
@@ -2799,12 +2799,7 @@ class BaseCurrentLoopTests:
     def setUp(self):
         super().setUp()
         self.loop = asyncio.new_event_loop()
-        asyncio.set_event_loop(self.loop)
-
-    def tearDown(self):
-        self.loop.close()
-        asyncio.set_event_loop(None)
-        super().tearDown()
+        self.set_event_loop(self.loop)
```

Comment:
@asvetlov: I'm not sure about this change, but since tests still pass, it seems like the change is safe.